R/eval-select.R

```diff
@@ -43,6 +43,8 @@
 #'   use predicates (i.e. in `where()`). If `FALSE`, will error if `expr` uses a
 #'   predicate. Will automatically be set to `FALSE` if `data` does not
 #'   support predicates (as determined by [tidyselect_data_has_predicates()]).
+#' @param error_arg Argument name to include in error message if `allow_empty = FALSE`.
```

Comment:
ideally, it should never resolve to being multiple arguments. Since `before` and `after` being both supplied should result in another error message..  (can't supply `after` and `before`

Reply:
oh yeah, makes sense!